pennylane_lightning_gpu/lightning_gpu.py

```diff
@@ -142,6 +142,7 @@ def _mebibytesToBytes(mebibytes):
     "Identity",
     "BasisState",
     "QubitStateVector",
```

Comment:
are we going to remove QSV here?

Reply:
It's what I thought, but it should be supplemented, not replaced (see Tom's comment https://github.com/PennyLaneAI/pennylane-lightning-kokkos/pull/79#discussion_r1301585672).